src/Tenancy.php

```diff
@@ -85,6 +85,27 @@ public function getBootstrappers(): array
         return array_map('app', $resolve($this->tenant));
     }
 
+    /**
+     * @param callable $callback
+     * @return mixed
+     */
+    public function runGlobal(callable $callback)
```

Comment:
I like to use `global` for things that are genuinely global — things that are the same in the central & tenant parts of the app.

So `central` is more appropriate here.

I think `tenancy()->central(callable $callback)` is the ideal solution.

Reply:
As I mentioned in the description of this pr i went back and furth on this.. We could settle on global if you prefer.. 👍

Reply:
Method name has been renamed